tests/get-support.spec.ts

```diff
@@ -0,0 +1,40 @@
+import { OnboardingSelectors } from '@tests-legacy/integration/onboarding/onboarding.selectors';
+import { SettingsSelectors } from '@tests-legacy/integration/settings.selectors';
+import { FundPageSelectors } from '@tests-legacy/page-objects/fund.selectors';
+
+import { RouteUrls } from '@shared/route-urls';
+
+import { test } from './utils/testfn';
+
+test('get support menu item takes user to support page', async ({ page, extensionId }) => {
+  await page.route(/.*/, route => route.abort());
+  await page.route(/chrome-extension/, route => route.continue());
```

Comment:
what does this do?

Reply:
Since all network requests are blocked on line 10, line 11 overrides and allows requests to the extension's assets to continue. 

This is something we'll most likely abstract into a fixture as we define our conventions.